src/components/RequestBox/RequestBox.js

```diff
@@ -412,14 +417,12 @@ export default class RequestBox extends Component {
       body: serializer.serializeToString(newRx)
     })
       .then(response => {
-        console.log('sendRx response: ');
+        console.log('Successfully sent NewRx to PIMS');
         console.log(response);
-        this.props.consoleLog('Successfully sent Rx to PIMS', types.info);
+        this.handleRxResponse();
```

Comment:
Should this check the return code, or the data that was returned to see if it was a valid RxStatus message returned?